octodns/provider/cloudflare.py

```diff
@@ -85,11 +97,27 @@ def __init__(self, id, email=None, token=None, cdn=False, *args, **kwargs):
                 'Authorization': 'Bearer {}'.format(token),
             })
         self.cdn = cdn
+        self.retry_count = retry_count
+        self.retry_period = retry_period
         self._sess = sess
 
         self._zones = None
         self._zone_records = {}
 
+    def _try(self, *args, **kwargs):
```

Comment:
Minor, but what do you think about calling this `_try_request`? It seems a little more descriptive of what it's doing since `_try` is super generic.

Reply:
Sure, incoming update